packages/commonwealth/client/scripts/views/pages/CommunityManagement/Contests/ContestsList/ContestCard/ContestCard.tsx

```diff
@@ -119,7 +147,9 @@ const ContestCard = ({
               <CWText className="label">
                 {moment.localeData().ordinal(index + 1)} Prize
               </CWText>
-              <CWText fontWeight="bold">{s.tickerPrize} ETH</CWText>
+              <CWText fontWeight="bold">
+                {capDecimals(s.tickerPrize!.toFixed(decimals))} {ticker}
```

Comment:
`decimals` is an optional prop, lets check if it exists before applying `.toFixed()`

Reply:
added default value. It's optional because we have broken trpc types :/